src/main/java/com/minecolonies/structures/helpers/StructureProxy.java

```diff
@@ -365,23 +263,35 @@ public void rotate(final int times)
         minY = Math.abs(minY);
         minZ = Math.abs(minZ);
         boolean foundHut = false;
+        PlacementSettings settings = new PlacementSettings().setRotation(rotation);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'settings' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)